src/RestApiClient.js

```diff
@@ -162,26 +163,49 @@ class RestApiClient {
 
     merge(options, data);
 
-    // Developer Output:
-    console.groupCollapsed('%c Gateway: %c %s %s %s',
-      'background-color: rgba(105,157,215,0.5); color: rgba(33,70,112,1); display: block; font-weight: bold; line-height: 1.5;',
-      'background-color: transparent; color: black; font-weight: bold; line-height: 1.5;',
-      method,
-      url.host,
-      url.pathname
-    );
-    console.log('URL: %s', url.toString());
-    console.log('Options:', options);
-    console.groupEnd();
+    gatewayLog(method, url, options);
 
     return window.fetch(url, options)
       .then(this.checkStatus)
       .then(this.parseJson)
-      .catch((data) => {
-        console.log(data);
-        console.error('um, there was an error! handle that shizzzzz!');
+      .catch((error) => {
+        gatewayError(error);
+
+        throw error;
```

Comment:
This also drove me nuts, but less-so than the `multipart/form-data` content type issue. 🤕